powernet-project/powernet
# UI updates

# Current issues
1. sidebar covers the main content when toggled

# Code changes
1. removed vertical scroll on the sidebar
2.1 made dashboard on user page to look more like the initial mockups
2.2 removed the js used for previous version
3. changed icon size in main farm page
4. added border to top bar buttons in main home page

# Documentation changes
none

# Validation suggestions
1. log in with farm account and click overview to see the change in icon size
2. log in with home account to view the dashboard page
3. test sidebar on either acount
